react-todo-app/src/App.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import TodoForm from './components/TodoForm';
+import TodoInfoList from './components/TodoInfoList';
+
+class App extends Component {
+
+  // id를 state로 관리하지 않는 이유는 id는 렌더링되는 정보가 아니기 때문이다.
+  id = 0;
+
+  // App.js가 onCreate를 props로 TodoFrom.js에 전달해 데이터들을 받아왔으니  
+  // 이를 배열을 이용해서 저장해준다. 
+  state = {
+    information: [],
+  }
+
+  handleCreate = (data) => {
```

Comment:
In this code, when you define a method, you define the function as a property of the class rather than writing it as a class's method type. You can do this, but the following article will help you. In summary, defining a function with a class property can cause performance problems. This review should be a good reference only.

JFYI : https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1